sync.js

```diff
@@ -18,6 +17,17 @@ const SYNCFILE_FORMATS = {
   'osm-p2p-syncfile'   : 2
 }
 
+const WIFI_READY = 'replication-wifi-ready'
+const REPLICATION_PROGRESS = 'replication-progress'
+const REPLICATION_COMPLETE = 'replication-complete'
+const REPLICATION_ERROR = 'replication-error'
+const REPLICATION_WAITING = 'replication-waiting'
+
+function OpenConnection (target, topic, message) {
```

Comment:
`OpenConnection` feels.. misleading? It's not really opening a connection. Maybe `setTargetState`?

Reply:
cool i was thinking like it is tracking the currently open connections, states sound fine though

Reply:
It's tracking the topic and message though right, not a connection? Am I misunderstanding?

Reply:
yeah changed